### ISSUE TITLE
[VSLCV] radio buttons overlap "next" button in tutorial

### ISSUE THREAD

**Issue by adamamiller** [zooniverse/front-end-monorepo#1852]
**Package**

VSLCV

**Is your feature request related to a problem? Please describe.**

The "next" button in the tutorial overlaps the radio buttons that show the "progress" through the tutorial. This makes it impossible to advance all the way through the tutorial, since the next button effectively does not work. 

![image](https://user-images.githubusercontent.com/2434744/96194607-5717c100-0f10-11eb-9bbf-74b13ef068aa.png)

**Describe the solution you'd like**
Is it possible to show a smaller subset of radio buttons? I realize this may not be possible.

**Describe alternatives you've considered**
A shorter tutorial would fix this, but given the large array of new features that are new to this project requires a lot of explanation. 

**Additional context**
This is probably not critical for the beta test as most users will probably complete the tutorial in the initial pop-up window.

**Comment by trouille** [comment]
@adamamiller You may have missed it, but in https://www.zooniverse.org/projects/adamamiller/stellar-sleuths/talk/3790/1633842 I shared our best practices around Tutorials. 

"Also, here's a helpful resource for how to write an effective Zooniverse tutorial: https://blog.zooniverse.org/2019/06/13/top-ten-tips-writing-a-great-zooniverse-tutorial/

For your project, the balance between giving full instructions and details versus people actually reading the whole tutorial is going to be the trickiest part. I encourage sharing your tutorial with your partner or others to get their feedback.

General rule of thumb: Most tutorials are no longer than 5 or 6 slides and no more than a few hundred characters on each slide.

Gifs/images, as you're planning, can be very helpful in conveying information."

Please reduce the number of slides -- we know from experiments that there's a drop off in the number of people who click to slides after 5 slides. Some projects have 7 or 8, but it's strongly recommended to keep with 5 or 6.

**Comment by adamamiller** [comment]
@trouille I did see the note earlier! But thank you for bringing this our attention once again. We have managed to get everything down to 7 slides. I'll try to reduce the word count some as well. 

@srallen One thing I've realized in reducing the size of the tutorial is that I'd like to link to the FAQ under the About section within the Tutorial. Is that something that will have to wait until you link everything up on your end?
